src/diffusers/schedulers/scheduling_ddpm.py

```diff
@@ -194,7 +194,7 @@ def __init__(
         sample_max_value: float = 1.0,
         timestep_spacing: str = "leading",
         steps_offset: int = 0,
-        rescale_betas_zero_snr: int = False,
+        rescale_betas_zero_snr: bool = False,
```

Comment:
You might have to run `make fix-copies` to fix all instances of this type hint. If you find more inconsistencies, please feel free to open follow-up PRs!

Reply:
Thank you @a-r-r-o-w for the information!